Dockerfile.api

```diff
@@ -16,25 +16,27 @@ RUN python -m textacy download lang_identifier --version 3.0 \
     && python -m spacy download es_core_news_md \
     && python -m spacy download fr_core_news_md
 
+COPY . .
+
 #####
 FROM base AS dev
 
 RUN python -m pip install -r requirements/dev.txt
 
-COPY . .
-
```

Comment:
Just checking, why move this? I was hoping to leverage Docker's caching, since requirements files change less often than the source code itself.